src/style.css

```
@@ -0,0 +1,11 @@
+.input_margin {
```

Краще не вказувати в класі, що він робить
```suggestion
.input {
```

---

запомню, спасибо!